.github/workflows/trigger_benchmarks.yml

```diff
@@ -0,0 +1,100 @@
+name: Trigger Benchmarks
+on:
+  issue_comment:
+    types:
+      - created
+
+env:
+  BENCHMARK_TRIGGER_COMMAND: /benchmark
+
+jobs:
+  trigger_benchmark_webhook:
+    runs-on: ubuntu-latest
+
+    # Only run for pull request comments (newly created)
+    if: github.event.issue.pull_request
+
+    steps:
+      - name: Check Comment Body
+        id: comment_body
+        run: echo "triggers_benchmark=${{ github.event.comment.body == env.BENCHMARK_TRIGGER_COMMAND }}" >> $GITHUB_OUTPUT
+
+      - name: Set Repository Info
+        id: repo_info
+        env:
+          REPO_OWNER: ${{ github.repository_owner }}
+        run: |
+          echo "REPO_OWNER=$REPO_OWNER" >> $GITHUB_OUTPUT
+          echo "REPO_NAME=${GITHUB_REPOSITORY#$REPO_OWNER/}" >> $GITHUB_OUTPUT
```

Comment:
`GITHUB_REPOSITORY` hasn't been set as an env var.

Reply:
Hey Mudit, `GITHUB_REPOSITORY` is set by GitHub Actions itself and is available to all shell scripts within actions (it is the equivalent to doing `${{ github.environment }}`

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
![image](https://github.com/user-attachments/assets/9b9f51ab-f66d-4606-99f4-c8ffd464e543)

Reply:
Nice! Thanks for the info 😄